submodules/command_ingest/__init__.py

```diff
@@ -1,114 +1,102 @@
 import logging
+from inspect import signature
+from collections import deque as queue
+
+# Registered commands
+# import core
+# from submodules import aprs, iridium, telemetry
 
 logger = logging.getLogger("CI")
 
 total_received: int = 0
 total_errors: int = 0
 total_success: int = 0
 
-# All currently registered commands.  The key is the function identifier in the air, and values follow
-# the format {"function": func, "args": args}  where args is an array of types (e.g. int, str, float, etc.)
-registered_commands = {}
 
+general_queue = queue()
```

Comment:
Do not initialize variables outside a function because they will be reinitialized everytime the file is imported. Instead move these variable defs to the `start()` function